src/gnssro/gnssro_bufr2ioda.py

```diff
@@ -39,13 +40,32 @@
     ("dateTime", "long")
 ]
 
+# Data used in fill_missing_satellite_subidentifier from file
+# names that use satDAAC identifiers, like the UCAR/COSMIC archive.
+# The first parentheses in the pattern should match the satId
+# and the second parentheses should match the number (subidentifier).
+# Satellite subidentifiers are typically not needed for government
+# missions. For example, Cosmic1 (C[0-9]{3}) and Cosmic2 (C2E[0-9])
+# do not use subidentifiers.
+satIdPatterns = [
+    r'^(GN)([0-9]{2})$',     # PlanetiQ
+    r'^(S)([0-9]{3})$',      # Spire
+    r'^(GO)([0-9]{2})$'      # GeoOptics
+]
 
-def main(args):
+satIdLists = [
+    [267, 268],             # PlanetiQ
```

Comment:
```suggestion
    [267, 268, 768],             # PlanetiQ
```

you mean like this @steve-marshall-piq ?  I think you can make a suggestion :0)

is this the only location a change is needed otherwise I think this is fine it should leave all the old files from GRAS, TanDEM-X, KOMPSAT-5, etc alone it does run through the new `fill_missing_satellite_subidentifier` function but only modifies the `profile_meta_data` is a match is found

Reply:
@BenjaminRuston - This looks good.  768 is an additional satelliteIdentifier associated with PlanetiQ.  It is used for the latest generation of PlanetiQ satellites (the "YAM" series).

You are correct that fill_missing_satellite_subidentifier will not do anything unless the satellite subidentifier is not found in the BUFR data, and the subidentifier can be found in the filename of the BUFR data.